src/common/Configuration/Config.h

```diff
@@ -51,16 +51,16 @@ class ConfigMgr
      * Deprecated geters. This geters will be deleted
      */
 
-    // @deprecated DO NOT USE - use GetOption<std::string> instead.
+    [[deprecated("Use GetOption<std::string> instead")]]
```

Comment:
Good use.

But doesnt this have to be directly infront of the function?

Reply:
Check ci